SEM 2 PRACTICALS/prac_05/emails.py

```diff
@@ -0,0 +1,50 @@
+"""
+CP1404/CP5632 Practical
+emails program
+"""
+
+
+def main():
+    emails = []
```

Comment:
Why are you doing this? Where is the dictionary?
Prac 5 is all about dictionaries: 
> Write a program that stores users' emails (unique keys) and names (values) in a dictionary.